src/components/DataBrowser.vue

```diff
@@ -207,12 +207,12 @@ v-data-table.girder-file-browser-component.elevation-1(
   //- Table Row Slot
   template(slot="items", slot-scope="props")
     tr.itemRow(:active="props.selected",
-        @click="props.selected = !props.selected",
+        @click="if (selectEnabled) props.selected = !props.selected;",
```

Comment:
Sometimes, it's better not even bind some event because some Vuetify component changes behavior when there is a handler. But it should be fine in this case, and I can't think of a cleaner way of doing so.